### ISSUE TITLE
[Bug]: Ice cannot be configured to allow Little Snitch Mini connection status to be visible in menu bar

### ISSUE THREAD

**Issue by KenGoBlue** [jordanbaird/Ice#145]
### Check existing issues

- [X] I have checked existing issues and believe that my issue is not a duplicate

### Description

It looks like Ice is always hiding Little Snitch Mini's menu bar status item. 

### Steps to Reproduce

1. Quit Ice.
2. Launch Little Snitch Mini and enable "Show status in menu bar"; verify that the connection status is shown.
3. Launch Ice; Little Snitch Mini's connection status disappears.
4. Cmd-click (or hover, or etc., depending on Ice settings) to view hidden items; Little Snitch Mini's connection status still isn't visible in the menu bar.
5. Toggle Ice's always-hidden section; Little Snitch Mini connection status still isn't visible in the menu bar.
6. Quit Ice; Little Snitch Mini connection status reappears.

### Ice Version

0.9.0

### macOS Version

14.5 (23F79)

**Comment by jordanbaird** [comment]
I can't reproduce this. Ice isn't able to remove items from the menu bar. Are you sure you're not confusing the hidden section with the always-hidden section? To get the always-hidden section to appear, Option+click the Ice icon.

**Comment by KenGoBlue** [comment]
I think I've found a way to make the menu bar item appear, but I'm not sure whether there's still an issue. I had originally tested this by 
1. Cmd-clicking the Ice menu bar item and toggling "Hide/Show the Always-Hidden Section."
2. Turning section dividers on to verify that I'm seeing the always-hidden section.
3. Moving other menu bar items in and out of the always-hidden section.

This time, I took the always-hidden section divider item farther to the left of any possibly hidden items, well to the other side of the notch, to make sure that section was empty; the Little Snitch Mini menu item reappeared. 

I'm not sure why it would have defaulted to the always-hidden section — but more importantly I'm not sure why it wasn't visible when the always-hidden section should have been shown. Possibly the notch was an issue? I'd thought I had Menu Bar Appearance and "Hide application menus when showing menu bar items" set to prevent that, tho'. Maybe a combination of the notch and the Little Snitch Mini menu bar item being slightly wider than other items? I'm not sure.
